src/uu/factor/src/numeric/montgomery.rs

```diff
@@ -113,7 +113,7 @@ impl<T: DoubleInt> Arithmetic for Montgomery<T> {
         self.n.as_u64()
     }
 
-    fn from_u64(&self, x: u64) -> Self::ModInt {
+    fn to_mod(&self, x: u64) -> Self::ModInt {
```

Comment:
The naming here is a bit confusing.

What about just using `mod()`, similar to the other `add()`, `pow()`, ... "operator" functions?

I do see the `modulus()` function which returns the current instance modulus, but I think that this shorter version used as an "operator" might be reasonable and make it more clear what the function is for future devs.

Thoughts?

Reply:
It's a conversion from `u64` to `Self::ModInt`, where `add()` and co. take a `Self::ModInt` and return a `Self::ModInt`. I had `to_modint` before, but then changed it to this. Would you prefer `to_modint` instead of `to_mod`?

Reply:
Ah, I mistakenly read the `r = ...` line as being self mod n, not converting n to a ModInt.

Yes, I think your original thought of `to_modint()` would be helpful to understand the code.